zulu-11/Dockerfile

```diff
@@ -4,13 +4,9 @@ FROM gcc:9.2.0 as dumb-init-builder
 RUN git clone -b v1.2.2 --depth 1 --single-branch https://github.com/proofpoint/dumb-init.git
 RUN cd dumb-init && make
 
-FROM docker.io/azul/zulu-openjdk-debian:11.0.7-11.39.15
+FROM docker.io/azul/zulu-openjdk-debian:11.0.8-11.41.23
 COPY --from=dumb-init-builder /dumb-init/dumb-init /usr/local/bin/dumb-init
 
-RUN apt-get update && \
-   DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes \
-       # These packages are security updates
-       libgnutls30=3.6.7-4+deb10u4 \
-       libsystemd0=241-7~deb10u4
+RUN apt-get update
```

Comment:
Why run apt-get update at this point?

Before we'd comment out the lines to make subsequent updates easier. See /f851d8f4ff5d6363de26ea29c757ae5ced4c3fd7